dmff/admp/settings.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from jax import jit
+from jax.config import config
+
+PRECISION = 'double'  # 'double'
+
+DO_JIT = True
+
+if PRECISION == 'double':
```

Comment:
Some of these settings are not admp-specific, and should be global.
Considering moving settings.py under dmff folder, instead of admp